app/models/manageiq/providers/redhat/network_manager.rb

```diff
@@ -0,0 +1,126 @@
+class ManageIQ::Providers::Redhat::NetworkManager < ManageIQ::Providers::NetworkManager
+  require_nested :CloudNetwork
+  require_nested :CloudSubnet
+  require_nested :EventCatcher
+  require_nested :EventParser
+  require_nested :MetricsCapture
+  require_nested :MetricsCollectorWorker
+  require_nested :NetworkPort
+  require_nested :RefreshParser
+  require_nested :RefreshWorker
+  require_nested :Refresher
+
+  include ManageIQ::Providers::Openstack::ManagerMixin
+  include SupportsFeatureMixin
+
+  supports :create
+
+  has_many :public_networks,  :foreign_key => :ems_id, :dependent => :destroy,
+           :class_name => "ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork::Public"
+  has_many :private_networks, :foreign_key => :ems_id, :dependent => :destroy,
+           :class_name => "ManageIQ::Providers::Openstack::NetworkManager::CloudNetwork::Private"
+
+  # Auth and endpoints delegations, editing of this type of manager must be disabled
```

Comment:
pls rephrase this message - it is a bit unclear

Reply:
Removed the comment. It was copied from the openstack network manager and not relevant here. Thanks.